### ISSUE TITLE
Wiki vs gh-pages

### ISSUE THREAD

**Issue by luispuerto** [OSGeo/homebrew-osgeo4mac#823]
I didn't want to continue the discussion in #822 because it wasn't the place for that. 

I think we have to decide if: 

- we keep wiki and gh-pages
- we keep just wiki 
- we keep just gh-pages

I think we should decide something like this because I see pointless to have the same info in both sites. 
We can test for a while in both but...

>As for the wiki, I think that using gh-page is better, you can add more sections and it's much more beautiful than the wiki.

I agree with you @fjperini  there that gh-pages it's much beautiful, versatile and so... However, I really don't know if a repo that is a tap is the best place for a gh-pages docs. I mean, a tap perhaps should be as clean as possible and usually a gh-pages is not that way. I've already thought about that the other day when I uploaded the logo for the readme and do that perhaps it wasn't the best think to have in the repo. Think that when a user make a `brew update` download a complete shallow clone of the tap. 

Brew has it's docs in the repo, but hombrew-core doesn't have anything there, just the formulae. However casks has stuff (just doc, not a website). 

Summing up, I like more the a website for the documentation... you are even able to search there while in the wiki you just can search names of pages.

I'm going to ponder this with the pillow..

**Comment by luispuerto** [comment]
PS/ on the other had the wiki is simple and it's  a "repo" insulated form the regular repo.

**Comment by luispuerto** [comment]
We decided to close the wiki por now 🙂
